src/main/java/org/lantern/LanternUtils.java

```diff
@@ -1371,4 +1371,15 @@ public static boolean isShuttingDown() {
         }
         return false;
     }
+
+    /**
+     * Utility for whether or not we're running on a Pi. Just checks for ARM 
+     * for now.
+     * 
+     * @return <code>true</code> if we're on ARM, otherwise <code>false</code>.
+     */
+    public static boolean isLanternPi() {
+        return LanternUtils.isDevMode() || 
+                SystemUtils.OS_ARCH.toLowerCase().contains("arm");
+    }
```

Comment:
The name of the method is kinda weird since this also returns true if we're running in dev mode.